api/napi/index.ts

```diff
@@ -45,7 +45,7 @@ interface ModelPeer {
  * A model is organized like a table with rows of data. The
  * fields of the data type T behave like columns.
  */
-interface Model<T> {
+export interface Model<T> {
```

Comment:
I personally think that in the "type" documentation here (for Model<T>) there should be a simple example of usage, including a reference to ModelPeer.

Reply:
done